valkey.conf

```diff
@@ -668,6 +668,22 @@ repl-diskless-sync-max-replicas 0
 #                 during replication.
 repl-diskless-load disabled
 
+# This dual channel replication sync feature optimizes the full synchronization process 
+# between a primary its replicas. When enabled, it reduces both memory and CPU load 
+# on the primary server.
+#
+# How it works:
+# 1. During full sync, instead of accumulating replication data on the primary server,
+#    the data is sent directly to the syncing replica.
+# 2. The primary's background save (bgsave) process streams the RDB snapshot directly
+#    to the replica over a separate connection.
+#
+# Benefits:
+# - Reduces memory usage on the primary server
+# - Decreases CPU load on the primary server's main process
+# - Improves overall synchronization efficiency
```

Comment:
Cut this, it's duplicate of the first paragraph. "Improves overall synchronization efficiency" <- I don't know what this means, seems like a reiteration of the first two points.

Reply:
👍